### PR TITLE
[OP-158] change certificate design

### PR DESCRIPTION
**Pull request by ihor-romaniuk**
**Description:** change certificate design

**Youtrack:** https://youtrack.raccoongang.com/issue/OP-158

### CODE REVIEW

File: lms/static/sass/certificate.scss [hunk 1]
```diff
@@ -204,62 +213,75 @@
...
-	}
+    text-align: center;
+    margin-top: 20px;
+
+    &-item {
+        display: inline-block !important;
+        vertical-align: top;
+        padding: 0 20px;
+        max-width: 215px;
+		width: 215px;
```
Comment (Lunyachek) [inline lms/static/sass/certificate.scss:243]: чето с отступом

File: lms/static/sass/certificate.scss [hunk 2]
```diff
@@ -204,62 +213,75 @@
...
+    img {
+        display: block;
+        position: absolute;
+        left: 50%;
+        bottom: 0;
+        transform: translateX(-50%);
+        height: auto;
+        width: auto;
+        max-width: 105px;
+		max-height: 95px;
```
Comment (Lunyachek) [inline lms/static/sass/certificate.scss:267]: и тут

**Review by Lunyachek — APPROVED** [review]

File: lms/static/sass/certificate.scss (hunk 1 above)
Comment (ihor-romaniuk) [reply lms/static/sass/certificate.scss:243]: fixed

File: lms/static/sass/certificate.scss (hunk 2 above)
Comment (ihor-romaniuk) [reply lms/static/sass/certificate.scss:267]: also fixed

**Review by vlasovmichael — APPROVED** [review]

File: lms/templates/certificates/_accomplishment-rendering.html [hunk 3]
```diff
@@ -5,37 +5,64 @@
...
+                    <strong>
+                        <img class="signatory-signature" alt="${_('T.S. Wojtiuk')}"
+                             src="/static/${settings.FEATURES.get('OP_SITE_THEME', settings.DEFAULT_SITE_THEME)}/images/signature.png">
+                    </strong>
+                    <span class="signatire-name"><b>${_("T.S. Wojtiuk")}</b></span>
+                    <span class="signatire-title">${_("Head of online learning department")}</span>
+                </div>
                 % if certificate_data:
                 % for signatory in certificate_data.get('signatories', []):
+                % if signatory['name'] or signatory['signature_image_path']:
```
Comment (arsentur) [inline lms/templates/certificates/_accomplishment-rendering.html]: Are you sure that these keys always exist?

File: lms/templates/certificates/_accomplishment-rendering.html [hunk 4]
```diff
@@ -5,37 +5,64 @@
...
                 <div class="signature-item">
-                    <strong><img class="signatory-signature" src="${static.url(signatory['signature_image_path'])}" alt="${signatory['name']}"></strong>
-                    <span>${signatory['name']}</span>
+                    <strong class="signature-image-wrapper">
+                        % if signatory['signature_image_path']:
+                        <img class="signatory-signature" src="${static.url(signatory['signature_image_path'])}" alt="${signatory['name']}">
```
Comment (arsentur) [inline lms/templates/certificates/_accomplishment-rendering.html]: Wrong indentation here and below

File: lms/templates/certificates/_accomplishment-rendering.html [hunk 5]
```diff
@@ -5,37 +5,64 @@
...
+                        % endif
+                    </strong>
+                     % if signatory['name']:
```
Comment (arsentur) [inline lms/templates/certificates/_accomplishment-rendering.html]: The same question about keys

File: lms/templates/certificates/_accomplishment-rendering.html [hunk 6]
```diff
@@ -5,37 +5,64 @@
...
+                    <span class="signatire-name"><b>${signatory['name']}</b></span>
+                    % endif
+                     % if signatory['name']:
```
Comment (arsentur) [inline lms/templates/certificates/_accomplishment-rendering.html]: Maybe we need to change it on `title`?

File: lms/templates/certificates/_accomplishment-rendering.html [hunk 7]
```diff
@@ -5,37 +5,64 @@
...
+                    <span class="signatire-title">${signatory['title']}</span>
+                    % endif
+                     % if signatory['name']:
```
Comment (arsentur) [inline lms/templates/certificates/_accomplishment-rendering.html]: ditto

File: lms/templates/certificates/_accomplishment-rendering.html (hunk 5 above)
Comment (arsentur) [inline lms/templates/certificates/_accomplishment-rendering.html]: Can we use here for loop with the key value pairs?

**Review by arsentur — CHANGES_REQUESTED** [review]

File: lms/templates/certificates/_accomplishment-rendering.html (hunk 6 above)
Comment (ihor-romaniuk) [reply lms/templates/certificates/_accomplishment-rendering.html]: of course, thanks

File: lms/templates/certificates/_accomplishment-rendering.html (hunk 5 above)
Comment (ihor-romaniuk) [reply lms/templates/certificates/_accomplishment-rendering.html]: we can not use a loop here

Comment (ihor-romaniuk) [reply lms/templates/certificates/_accomplishment-rendering.html]: yes

File: lms/templates/certificates/_accomplishment-rendering.html (hunk 3 above)
Comment (ihor-romaniuk) [reply lms/templates/certificates/_accomplishment-rendering.html]: fixed

File: lms/templates/certificates/_accomplishment-rendering.html (hunk 4 above)
Comment (ihor-romaniuk) [reply lms/templates/certificates/_accomplishment-rendering.html]: also

**Review by arsentur — APPROVED** [review]

**Pull request merged**